### PR TITLE
Gazette special facet

### PR DESCRIPTION
**Pull request by actlikewill**
- adds "special" filter facet for gazettes
- allows facets options to have separate name and value
image

### CODE REVIEW

File: peachjam/views/gazette.py [hunk 1]
```diff
@@ -122,6 +122,10 @@ class GazetteYearView(YearMixin, FilteredDocumentListView):
     navbar_link = "gazettes"
     locality = None
     group_by_date = "month-year"
+    special_facet_options = [
+        {"name": _("Special Issue"), "value": "special"},
```
Comment (longhotsummer) [inline peachjam/views/gazette.py:126]: the `_` must be the lazy variant, so that the translation is done at the last minute.

Comment (actlikewill) [reply peachjam/views/gazette.py]: is it okay to change it from the import thereby changing it everywhere in the file?

Comment (actlikewill) [reply peachjam/views/gazette.py]: I've seen we do it elsewhere so i guess its okay

Comment (longhotsummer) [reply peachjam/views/gazette.py]: > I've seen we do it elsewhere so i guess its okay

yes it shouldn't hurt

**Review by longhotsummer — APPROVED** [review]

**Comment by actlikewill** [comment]
I've added supplement to the facets 
image

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**